### ISSUE TITLE
Expose Cutoff and Resonance global Dexed paramters in menu

### ISSUE THREAD

**Issue by probonopd** [probonopd/MiniDexed#90]
Similar to (De)tune, we should expose the Cutoff and Resonance global Dexed paramters in the menu and save them in `performance.ini`.

![image](https://user-images.githubusercontent.com/2480569/163567756-44cff6f6-d929-4459-9300-72eb2778bb9f.png)

**Comment by rsta2** [comment]
I think this can be added. The respective methods are in the `Dexed` class. Will do so.

**Comment by rsta2** [comment]
Unfortunately it's not that simple. There is a define `USE_FX` in *PluginFx.cpp*, which probably needs to be enabled, to enable the filter. Also I do not know the parameter range for the following methods:

```
void setFilterCutoff(float cutoff);
void setFilterResonance(float resonance);
```

@dcoredump Can you give some clarification? Does the filter increase the load much?

**Comment by dcoredump** [comment]
I am using the filter inside MicroDexed. The `USE_FX` guard is present because I wanted to disable the complete FX stack in MicroDexed to be able to use older processors.

For a RPi enabling the EQ should not be a problem.

**Comment by dcoredump** [comment]
Just forgot: the values for the filters are 0.0f .. 1.0f.

**Comment by rsta2** [comment]
@dcoredump Thanks!

**Comment by rsta2** [comment]
Implemented in #93.

**Comment by probonopd** [comment]
Works! :+1: Thank you very much.
